com.gluonhq.scenebuilder.yml

```diff
@@ -0,0 +1,42 @@
+app-id: com.gluonhq.scenebuilder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+cleanup:
+  - /share/info/*
+  - /share/locale/*
+  - /share/man/*
+  - /share/runtime/locale/*
+  - 
+modules:
+  - name: SceneBuilder
+    buildsystem: simple
+    only-arches:
+    build-commands:
+    - mkdir -p deb-package export/share
+    - ar p SceneBuilder-18.0.0.deb data.tar.xz | tar -xJf - -C deb-package
+    - mv deb-package/opt /app/lib 
+    - install -Dm644 com.gluonhq.scenebuilder.metainfo.xml -t /app/share/metainfo/
+    - install -Dm755 scenebuilder.sh /app/bin/SceneBuilder
+    - install -Dm644 com.gluonhq.scenebuilder.desktop /app/share/applications/com.gluonhq.scenebuilder.desktop
```

Comment:
there is `deb-package/opt/scenebuilder/lib/scenebuilder-SceneBuilder.desktop` extracted so I don't think there should  be a .desktop on this repo and the upstream one should be reused, renamed and edited with `desktop-file-edit`

Reply:
Done,
Also I try for reuse icon but it's not include SVG
Please check now.